geode-core/src/main/java/org/apache/geode/distributed/AbstractLauncher.java

```diff
@@ -96,7 +96,10 @@
 
   protected final transient AtomicBoolean running = new AtomicBoolean(false);
 
-  // TODO: use log4j logger instead of JUL
+  /**
+   * @deprecated Please use Log4J 2 instead.
+   */
+  @Deprecated
```

Comment:
I think you could just delete this, unless there's a breaking API change that I'm missing.  The only implementations that use this are `LocatorLauncher` and `ServerLauncher` and you've got those fixed

Reply:
The issue is that AbstractLauncher is itself part of the User API. Since `logger` is `protected` I opted for the more conservative approach of deprecating it -- especially after an earlier removal goof on my part, in which I removed a public or protected method from one or both subclasses.

Reply:
A User could subclass AbstractLauncher or one of its subclasses and use `logger` directly from their code.

Reply:
Cool that all sounds reasonable!